src/components/select-project/selectProject.html

```diff
@@ -1,9 +1,23 @@
-<ng-form name="$ctrl.forms.selectProjectForm">
+<span ng-if="$ctrl.noProjectsCantCreate" class="no-projects-cant-create">
```

Comment:
&lt;div&gt; is probably a better choice here since this is a decent-sized block of markup.